docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx

```diff
@@ -0,0 +1,307 @@
+
+```jsx filename="app/ui/video.jsx"
+export function Video() {
+  return (
+    <video width="320" height="240" controls preload="none">
+      <source src="/path/to/video.mp4" type="video/mp4" />
+      <track
+        src="/path/to/captions.vtt"
+        kind="subtitles"
+        srclang="en"
+        label="English"
+      />
+      Your browser does not support the video tag.
+    </video>
+  )
+}
+```
+
+### Common `<video>` Tag Attributes
+
+| Attribute     | Description                                                                                               | Example Value                              |
+| ------------- | --------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| `src`         | Specifies the source of the video file.                                                                   | `<video src="/path/to/video.mp4"></video>` |
+| `width`       | Sets the width of the video player.                                                                       | `<video width="320"></video>`              |
+| `height`      | Sets the height of the video player.                                                                      | `<video height="240"></video>`             |
+| `controls`    | If present, it displays the default set of playback controls.                                             | `<video controls></video>`                 |
+| `autoplay`    | Automatically starts playing the video when the page loads. Note: Autoplay policies vary across browsers. | `<video autoplay></video>`                 |
+| `loop`        | Loops the video playback.                                                                                 | `<video loop></video>`                     |
+| `muted`       | Mutes the audio by default. Often used with `autoplay`.                                                   | `<video muted></video>`                    |
+| `preload`     | Specifies how the video is preloaded. Values: `none`, `metadata`, `auto`.                                 | `<video preload="none"></video>`           |
+| `playsInline` | Enables inline playback on iOS devices, often necessary for autoplay to work on iOS Safari.               | `<video playsInline></video>`              |
+
+> **Good to know**: When using the `autoplay` attribute, it is important to also include the `muted` attribute to ensure the video plays automatically in most browsers, and the `playsInline` attribute for compatibility with iOS devices.
+>
+> ```
+> <video autoplay muted playsInline></video>
+> ```
+
+For a comprehensive list of video attributes, refer to the MDN documentation [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attributes).
+
+### Video best practices
+
+- **Fallback Content:** When using the `<video>` tag, include fallback content inside the tag for browsers that do not support video playback.
```

Comment:
Do we need to mention this? Looks like support is very good. https://caniuse.com/video

Reply:
I think we should just because it's considered best practice. Sometimes it isn't necessarily the video element but also the video type